service/app/util/Constant.java

```diff
@@ -0,0 +1,34 @@
+package util;
+
+/**
+ * This class will keep all the constant values.
+ *
+ * @author Manzarul
+ */
+public class Constant {
```

Comment:
This class has very general purpose constants. Ideally, all general purpose constants should be in common-util project and module specific "constant" class should extend the common constants class and have its custom constant definitions.

Also, we should start using interfaces or enums for defining constants.